src/pages/Delivery/Outside/DetailAddress.tsx

```diff
@@ -0,0 +1,177 @@
+import { useState, useCallback } from 'react';
+import useMarker from './hooks/useMarker';
+import useNaverMap from './hooks/useNaverMap';
+import CloseIcon from '@/assets/Main/close-icon.svg';
+import ArrowDown from '@/assets/Payment/arrow-down-icon.svg';
+import ArrowGo from '@/assets/Payment/arrow-go-icon.svg';
+import Modal, { ModalContent, ModalFooter } from '@/components/UI/Modal';
+import BottomModal, {
+  BottomModalHeader,
+  BottomModalContent,
+  BottomModalFooter,
+} from '@/components/UI/Modal/BottomModal';
+
+export default function DetailAddress() {
+  const sample: [number, number] = [36.767, 127.284];
+  const DetailRequest: string[] = [
+    '문 앞에 놔주세요 (벨 눌러주세요)',
+    '문 앞에 놔주세요 (노크해주세요)',
+    '문 앞에 놔주세요 (벨X, 노크 X)',
+    '직접 받을게요 ',
+    '전화주시면 마중 나갈게요',
+  ];
+
+  const [bottomModalIsOpen, setBottomModalIsOpen] = useState<boolean>(false);
+  const [modalIsOpen, setModalIsOpen] = useState<boolean>(false);
+
+  const [selectRequest, setSelectRequest] = useState<string>('');
```

Comment:
어법상 이렇게 써야 맞는 것 같아요!

```suggestion
  const [selectedRequest, setSelectedRequest] = useState<string>('');
```

Reply:
수정하겠습니다!